src/handlers/assign/action.ts

```diff
@@ -75,10 +75,24 @@ export const closePullRequestForAnIssue = async (): Promise<void> => {
   const prs = await getOpenedPullRequestsForAnIssue(payload.issue.number, "");
   if (!prs.length) return;
   logger.info(`Opened prs for this issue: ${JSON.stringify(prs)}`);
+
   let comment = `These linked pull requests are closed: `;
+  let noPullRequestsComment = `Failed To close these pull requests: `;
+
+  let deleteCount = 0;
```

Comment:
Seems not needed.
In the above code, if(!pr.length) return;
So it won't be reached here.